dac-ansible/master.yml

```diff
@@ -2,11 +2,6 @@
 - hosts: all
   become: true
   tasks:
-  - name: configure default gateway
-    command: route add default gw 10.43.255.1
-    register: command_result
-    failed_when: "command_result.rc != 0 and ('SIOCADDRT: File exists' not in command_result.stderr)"
-    changed_when: "command_result.rc == 0"
```

Comment:
I certainly needed this for the default networks in the scripts here, although we should probably fix the configuration in vss to fix that.